forge-ai/src/main/java/forge/ai/AiController.java

```diff
@@ -1082,6 +1082,17 @@ public int compare(final SpellAbility a, final SpellAbility b) {
                 }
             }
 
+            // If both are permanent creature spells, prefer the one that evaluates higher
+            if (a1 == b1 && a.getApi() == ApiType.PermanentCreature && b.getApi() == ApiType.PermanentCreature) {
+                int evalA = ComputerUtilCard.evaluateCreature(a.getCardState().getCard());
```

Comment:
Nice observation, I would have also thought this would have been wired up but couldn't find a trace of it either

The problem here is `getCardState().getCard()` does not return another switched version of the host

Currently I only know as fix a temporary manual switch of the state (so you could probably add helper `evaluateCreature(SA)`)

Similar to how I did it here:
https://github.com/Card-Forge/forge/blob/6de7dd6bcc4f6be5357ff0cd32f3426caa4a65de/forge-ai/src/main/java/forge/ai/AiController.java#L737

Reply:
Or try and rewrite `evaluateCreature` to query CardState from param directly

Reply:
Oh, nice, thanks a lot for the hint! I'll see if I can figure it out later today :)

Reply:
Tried something similar to what you demonstrated, please take a look when able :)